src/FrontDoor/FrontDoor.Test/ScenarioTests/FrontDoorTests.ps1

```diff
@@ -287,11 +287,10 @@ function Test-FrontDoorEndpointCustomDomainHTTPS-FrontDoor
     [int]$counter = 0
     do 
     {
-       Wait-Seconds 600
```

Comment:
As discussed, wait only in case of Record.

Reply:
I will just revert for now while I look how to get that environment variable.